WEBPACK_UPDATE.md

```diff
@@ -0,0 +1,59 @@
+# Upgrading from angular-cli@1.0.0-beta.10 to the angular-cli@1.0.0-beta.11-webpack 
+
+To update `angular-cli` to the webpack preview, you must update both the global package, project's local package and your project files.
```

Comment:
This is the official update procedure for ember-cli itself actually. I don't think it's a good idea to have de-synced versions of global and local... plus without the global version your `ng new` will generate old projects.